### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by grnkvch**
1. Task: https://github.com/rolling-scopes-school/nodejs-aws-tasks/blob/main/task3-product-api/task.md
> Information about setup and testing see README.md
2. Links:
> BACKEND
> https://1ey1lyj9ng.execute-api.eu-west-1.amazonaws.com/dev/products/b823ab27-61f0-4205-9123-87a8480f0b4a  
> https://1ey1lyj9ng.execute-api.eu-west-1.amazonaws.com/dev/products/
>
>SWAGGER https://app.swaggerhub.com/apis-docs/grnkvh/product-service/0.0.1

> FRONTEND
> PR: https://github.com/grnkvch/nodejs-aws-fe/pull/2
> https://dszl4pcvftgd3.cloudfront.net/

3. Deadline 01.11.2020 23:59
4. Score: 11/ 11
  - [x] poiproduct-service serverless.yml contains configuration for 2 lambda functions, API is not working at all, but YAML configuration is correct
  - [x] The getProductsList lambda function returns a correct response (POINT1)
  - [x] The getProductsById AND getProductsList lambda functions return a correct response code (POINT2)
  - [x] Your own Frontend application is integrated with product service (/products API) and products from product-service are represented on Frontend. AND POINT1 and POINT2 are done.
  - [x] Async/await is used in lambda functions
  - [x] ES6 modules are used for product-service implementation
  - [x] Webpack is configured for product-service
  - [x] SWAGGER documentation is created for product-service
  - [x] Lambda handlers are covered by basic UNIT tests (NO infrastructure logic is needed to be covered) (You may use JEST)
  - [x] Lambda handlers (getProductsList, getProductsById) code is written not in 1 single module (file) and separated in codebase.
  - [x] Main error scenarious are handled by API ("Product not found" error, try catch blocks are used in lambda handlers).

### CODE REVIEW

**Comment by YakovGrischenko** [comment]
Все сделано курто, но я обратил внимание что путь к API отличается от того, которыйы просили сделать в базовой части таска "The requested URL should be /products " и "The requested URL should be /products/{productId}".В остальном у меня нет замечаний.

**Comment by grnkvch** [comment]
> Все сделано курто, но я обратил внимание что путь к API отличается от того, которыйы просили сделать в базовой части таска "The requested URL should be /products " и "The requested URL should be /products/{productId}".В остальном у меня нет замечаний.

Спасибо за комментарий! 
Поправил это, см. последний коммит.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+```yarn install```  
+
+> 4. Go to subdirectory 'product-service' and install it`s dependencies  
+```cd product-service/ && yarn install```
+
+### RUN TEST
+
+> To run tests you have to be at root directory of the repositiry  
+run command:  
+```yarn jest```
```
Comment (ilya-moskovtsev) [inline README.md:21]: You may describe how to deploy the whole project and deploy a single function.

**Review by ilya-moskovtsev — APPROVED** [review]
Well done.

**Pull request merged**